operator/src/main/helm/coherence-operator/dashboards/services-summary-dashboard.json

```diff
@@ -196,12 +200,20 @@
       ],
       "targets": [
         {
-          "expr": "avg(coherence_partition_assignment_service_node_count{cluster=~\"$ClusterName\"}) by (coherence_service, tag_HA_status) ",
+          "expr": "avg(coherence_partition_assignment_service_node_count{cluster=~\"$ClusterName\"}) by (coherence_service) ",
           "format": "table",
           "instant": true,
           "intervalFactor": 1,
           "legendFormat": "",
           "refId": "A"
+        },
+        {
+          "expr": "avg(coherence_partition_assignment_HA_status{cluster=~\"$ClusterName\"}) by (coherence_service)",
```

Comment:
avg is misleading for HA_status.   It is a single enum for entire cluster.  Luckily only one node has value. 
Unsure what happens when change in coordinator if there is period of 2 nodes looking like they have the value. we really want one and only one HA_status per cluster, coherence_service.

Reply:
Hi @jfialli , the reason for having avg is to allow me to group by coherence_service and therefore making integrating this metric easier into the existing panel. Having always being only one value for cluster and service, the avg is not really giving us an average.
Overall doing it this way makes it easier to fit the metric into the panel.
Hope this makes sense.

Reply:
Thanks for explaining it.